src/display/api.js

```diff
@@ -879,8 +948,8 @@ class PDFDocumentProxy {
  * @property {boolean} [renderInteractiveForms] - Whether or not
  *                     interactive form elements are rendered in the display
  *                     layer. If so, we do not render them on canvas as well.
- * @property {Array}  [transform] - Additional transform, applied
- *                    just before viewport transform.
+ * @property {Array<any>}  [transform] - Additional transform, applied
```

Comment:
Interesting - I tried out https://www.npmjs.com/package/eslint-plugin-jsdoc to format all jsdoc, and the above is in fact the default output of `eslint-plugin-jsdoc`...

Anyway - removing the space for now. Perhaps it might be worth adding the `eslint-plugin-jsdoc` to the tools?